.github/ISSUE_TEMPLATE/story-issue-template.md

```diff
@@ -62,6 +62,14 @@ Required per [CM-4](https://nvd.nist.gov/800-53/Rev4/control/CM-4).
 [comment]: # "Please do not remove this section without care."
 [comment]: # "Note any security concerns that might be implicated in the change. 'None' is OK, but we must be explicit here."
 
+# Sketch
+
+We're thinking we'll try implementing A and then changing B.
+
+```[tasklist]
```

Comment:
What do you think about adding a short comment to describe what a "sketch" is for?

```suggestion
We're thinking we'll try implementing A and then changing B.

[comment]: # "Broad list of implementation steps that aren't as well-defined or 'demoable' as ACs"
```[tasklist]
```